rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/model/ArchiveCoordinateModel.java

```diff
@@ -0,0 +1,47 @@
+package org.jboss.windup.rules.apps.java.archives.model;
+
+import org.jboss.forge.addon.dependencies.Coordinate;
+import org.jboss.windup.graph.model.WindupVertexFrame;
+
+import com.tinkerpop.frames.Property;
+import com.tinkerpop.frames.modules.typedgraph.TypeValue;
+
+/**
+ * Represents a {@link Coordinate} for an {@link IdentifiedArchiveModel}.
+ *
+ * @author Ondrej Zizka, ozizka at redhat.com
+ * @author <a href="mailto:lincolnbaxter@gmail.com">Lincoln Baxter, III</a>
+ */
+@TypeValue(ArchiveCoordinateModel.TYPE)
+public interface ArchiveCoordinateModel extends WindupVertexFrame
```

Comment:
I am a little bit concerned about us not having a "type" field here. But maybe it is unnecessary... I guess the vast majority of artifacts to skip will be of type "jar".